Crane/app/src/main/java/androidx/compose/samples/crane/home/SearchUserInput.kt

```diff
@@ -72,8 +74,10 @@ fun PeopleUserInput(
         val tint = tintPeopleUserInput(transitionState)
 
         val people = peopleState.people
+        val resources = LocalContext.current.resources
+        val peopleString = resources.getQuantityString(R.plurals.number_adults_selected, people, people, titleSuffix)
```

Comment:
Plural support was added in the latest version of Compose, good place to try it!

Reply:
Cool @bentrengrove, think it'll be a good place for me to also to try update the docs then? https://developer.android.com/jetpack/compose/resources#string-plurals

Reply:
Definitely!

Reply:
Updated this PR with the change. [d5c2732](https://github.com/android/compose-samples/pull/765/commits/d5c2732bec52ef0c0a91969eed8b0aa80119ae8f)